src/Parser/TemplateTagParser.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDocParser\Parser;
+
+use PHPStan\PhpDocParser\Ast;
+use PHPStan\PhpDocParser\Ast\PhpDoc\TemplateTagValueNode;
+use PHPStan\PhpDocParser\Lexer\Lexer;
+
+class TemplateTagParser
+{
+
+	/**
+	 * @throws ParserException
+	 * @param ?callable(TokenIterator): string $parseDescription
+	 */
+	public static function parseTemplateTagValue(
```

Comment:
This can be a public instance method in TypeParser, right? That way it can be called from TypeParser and from PhpDocParser (which already contains TypeParser).

Reply:
you are right, it can :sweat_smile: got tunnel visioned by the comment on the old pr